core/src/main/scala/kanaloa/reactive/dispatcher/queue/Autothrottler.scala

```diff
@@ -103,56 +101,90 @@ trait Autothrottler extends Actor with ActorLogging with MessageScheduler {
   }
 
   private def optimize(currentSize: PoolSize): ScaleTo = {
+    val newSize = Autothrottler.optimize(currentSize, perfLog, settings)
+    ScaleTo(newSize, Some("optimizing"))
+  }
 
-    val adjacentPerformances: PerformanceLog = {
+  private def explore(currentSize: PoolSize): ScaleTo = {
+    val change = Math.max(1, Random.nextInt(maxExploreStepSize))
+    ScaleTo(
+      currentSize + (if (random.nextDouble() < chanceOfScalingDownWhenFull) -change else change),
+      Some("exploring")
+    )
+  }
+
+  private implicit def durationToJDuration(d: FiniteDuration): JDuration = JDuration.ofNanos(d.toNanos)
```

Comment:
I am going to bring the `JavaDurationConverters` to the kanaloa util package and use that instead